src/components/ebay-carousel/style.less

```diff
@@ -148,6 +148,16 @@
  * Not all browsers that have implemented scroll snapping have done so in a way that we can leverage.
  * Currently the below media query ensures that the browser supports exactly what we need.
  */
+
+@supports (-moz-appearance:none) {
+    // Explicitly disable support in firefox since we can't easily hide the scrollbar.
+    .carousel__list {
+        &&&& {
```

Comment:
Why is this repeated in this way? Is that a FireFox feature (bug) to actually match this selector?

Reply:
It is basically just `.carousel__list.carousel__list.carousel__list.carousel__list`. I'm just forcing a certain specificity instead of using `!important` so that it can be overwritten if needed.